src/abc/abc272/a.py

```diff
@@ -0,0 +1,21 @@
+def main():
+    # -----------
+    #    入力
+    # -----------
+
+    n = int(input())
+    a_list = list(map(int, input().split()))
+
+    # -----------
+    #    計算
+    # -----------
+
+    ans = 0
+    for a in a_list:
+        ans += a
```

Comment:
```suggestion
ans = sum(a_list)
```
参考: https://docs.python.org/ja/3/library/functions.html?highlight=sum#sum

Reply:
bad244d で対応